.travis.yml

```diff
@@ -87,6 +87,8 @@ jobs:
                 --volume=$HOME/.ccache-gcc-4.8:/.ccache:rw
                 --env CC=gcc-4.8
                 --env CXX=g++-4.8
+                --env CXXFLAGS=-m32
```

Comment:
As Werror is disabled in debugflags.cmake and we explictly enable it in build.py (which is not used here), we should add it here too to catch actual build errors.

Reply:
Also we are not doing a Debug build here, apparently. Not sure if this is really desired?

Reply:
Not sure about that. For me it's important that herbstluftwm builds for people with old systems. Hence, I wouldn't care so much about `-Werror` and debug build.

Reply:
I tend to agree with @t-wissmann here. On the other hand, it probably doesn't hurt either?

Reply:
In any case, it'd be separate PR. :-)

Reply:
Ok then!